src/node_buffer.cc

```diff
@@ -1129,14 +1129,16 @@ void SlowIndexOfNumber(const FunctionCallbackInfo<Value>& args) {
 }
 
 int32_t FastIndexOfNumber(v8::Local<v8::Value>,
-                          const FastApiTypedArray<uint8_t>& buffer,
+                          v8::Local<v8::Object> buffer,
                           uint32_t needle,
                           int64_t offset_i64,
                           bool is_forward) {
-  uint8_t* buffer_data;
-  CHECK(buffer.getStorageIfAligned(&buffer_data));
+  HandleScope handle_scope(buffer->GetIsolate());
```

Comment:
What does this do?

Reply:
I'm not sure it's necessary since no Local object are created in this scope? @addaleax

Reply:
When I didn't do it it crashed

Reply:
hmm, if it crashes without a handle scope, we should consider doing it in other fast APIs too..